packages/block-library/src/style.scss

```diff
@@ -24,6 +24,7 @@
 @import "./text-columns/style.scss";
 @import "./verse/style.scss";
 @import "./video/style.scss";
+@import "./spacer/style.scss";
```

Comment:
_Really_ minor note, but this list is arranged alphabetically. Can you move this line up to the correct alphabetical spot?

Reply:
Haha, totally! Thanks for catching that one.